src/routes/home.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  FastifyPluginAsyncTypebox,
+  Type,
+} from "@fastify/type-provider-typebox";
+
+const plugin: FastifyPluginAsyncTypebox = async (fastify) => {
+  fastify.get(
+    "/",
+    {
+      schema: {
+        response: {
+          200: Type.Object({
+            message: Type.String(),
+          }),
+        },
+        tags: ["Home"],
+      },
+    },
+    async function () {
+      return { message: "Welcome to the official fastify demo!" };
+    },
+  );
+};
```

Comment:
Please add a post route that validates the body.

Reply:
I added to `/example`